### PR TITLE
Discuss database schema

### PR DESCRIPTION
**Pull request by nisanthchunduru**

### CODE REVIEW

File: database_schema_and_api.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+### Movies
+
+title
+poster
+
```
Comment (gigimai) [inline database_schema_and_api.md:9]: should we add language here as well, we are supporting expats, and they don't wish to go for a movie with vietnamese dub? Right?

File: database_schema_and_api.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+### Theatres
```
Comment (gigimai) [inline database_schema_and_api.md]: `Theater`

File: database_schema_and_api.md (hunk 1 above)
Comment (gigimai) [inline database_schema_and_api.md:9]: And format, is it going to be in 2D or 3D

File: database_schema_and_api.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+name
+
+### Screenings
+
```
Comment (gigimai) [inline database_schema_and_api.md:24]: `Screening`

**Comment by gigimai** [comment]
I think we should go with a `screeningList` in the return data. What do you think?

File: database_schema_and_api.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+poster
+
+### Theatres
+
+Theatres are the places where movies are played. A theatre can have a name etc.
+
+#### Attributes
+
+name
+
```
Comment (gigimai) [inline database_schema_and_api.md:18]: address as well

**Comment by nisanthchunduru** [comment]
@gigimai By return data, do you mean the API response?

**Pull request merged**